internal/cypress/config.go

```diff
@@ -176,9 +178,31 @@ func checkAvailability(path string, mustBeDirectory bool) error {
 }
 
 // ValidateCypressConfiguration validates that Cypress config has required folders.
-func ValidateCypressConfiguration(configFile string) error {
-	configDir := filepath.Dir(configFile)
-	cfg, err := ConfigFromFile(configFile)
+func ValidateCypressConfiguration(rootDir string, cypressCfgFile string) error {
+	// cypress.json must be under rootDir
+	found, err := isCypressCfgUnderRootDir(rootDir, cypressCfgFile)
+	if err != nil {
+		return err
+	}
+	if !found {
+		return fmt.Errorf("cypress config file(%s) must be placed under rootDir(%s)", cypressCfgFile, rootDir)
+	}
```

Comment:
Hm, it doesn't look like this is really necessary. You are already loading the cypress config at https://github.com/saucelabs/saucectl/pull/402/files#diff-fd0498ecf67c49a4232d5f94227f2514bfe9bae7254241fd53020d259aff32d1R199. You can simply add that error information there and save yourself the double check :)

Reply:
This is a very edge case. Using following config and project, it will fail https://app.saucelabs.com/tests/dca514a497644c2aa4be5f96dc3f36a5
```
rootDir: tests/e2e
configFile: ../cypress.json
cypress project: tests/cypress/integration/blahblah
```
although seems to be very rare.